packages/replay-next/components/inspector/properties/FunctionLocationRenderer.tsx

```diff
@@ -0,0 +1,61 @@
+import { Object as ProtocolObject } from "@replayio/protocol";
+import classNames from "classnames";
+import { useContext } from "react";
+
+import { InspectorContext } from "replay-next/src/contexts/InspectorContext";
+import { SourcesContext } from "replay-next/src/contexts/SourcesContext";
+import { sourcesByIdCache } from "replay-next/src/suspense/SourcesCache";
+import { getSourceFileName } from "replay-next/src/utils/source";
+import { getPreferredLocation } from "replay-next/src/utils/sources";
+import { ReplayClientContext } from "shared/client/ReplayClientContext";
+
+import styles from "./shared.module.css";
+
+type FunctionLocationRendererProps = {
+  object: ProtocolObject;
+};
+
+export default function FunctionLocationRenderer({ object }: FunctionLocationRendererProps) {
+  const client = useContext(ReplayClientContext);
+  const { inspectFunctionDefinition } = useContext(InspectorContext);
+  const { preferredGeneratedSourceIds } = useContext(SourcesContext) ?? {};
```

Comment:
This change looks unexpected. Do we know which instances `SourcesContext` is null?

The `SourcesContextAdapter` (which renders the `SourcesContext` provider) is wrapped around the root of the `DevTools` component's children, right beneath the `SessionContext`.

I'd like to figure this out before we mrege.